pkg/certs/cert-inspection/certgraphanalysis/analyzer.go

```diff
@@ -6,13 +6,18 @@ import (
 	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
 	certificatesv1 "k8s.io/api/certificates/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/util/cert"
 )
 
 func InspectSecret(obj *corev1.Secret) (*certgraphapi.CertKeyPair, error) {
 	resourceString := fmt.Sprintf("secrets/%s[%s]", obj.Name, obj.Namespace)
 	tlsCrt, isTLS := obj.Data["tls.crt"]
 	if !isTLS {
+		if detail, err := InspectSecretAsKubeConfig(obj); err == nil {
+			return detail, nil
+		}
```

Comment:
Would it be useful to somehow log that this has been attempted and an error occurred?

Reply:
Any error is treated as critical here, so "collect TLS artifacts" test would fail if errors found. Also, any valid certificate would be treated as invalid kubeconfig, so we should continue instead of returning error.